<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="idleBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Idle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="survBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Surveillance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trckBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="engageBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Engagment</string>
    </property>
   </widget>
   <widget class="VideoGLWidget_gl" name="m_videowidget" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>741</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="trackIdListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>131</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 97, 203);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemperatureAlarm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>163</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblTemperatureAlarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelfCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>btnSelfCheck</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblBiasVoltageFault">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblBiasVoltageFault</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSystemStatus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>163</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblSystemStatus</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVersion">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblVersion</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDistance">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblDistance</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCounterMalfunction">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>163</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblCounterMalfunction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetFrequency">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>btnSetFrequency</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentFrequency">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>163</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblCurrentFrequency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuerySettingValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>163</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>btnQuerySettingValue</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblEchoStatus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>560</y>
      <width>163</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblEchoStatus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContinuousRanging">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>btnContinuousRanging</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblFrequency">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>78</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblFrequency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSingleRanging">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>btnSingleRanging</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQueryLaserCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>163</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>btnQueryLaserCount</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopRanging">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>340</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>btnStopRanging</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinFrequency">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblLaserCount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lblLaserCount</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoGLWidget_gl</class>
   <extends>QWidget</extends>
   <header location="global">videoglwidget_gl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
